packages/rocketchat-i18n/i18n/pt-BR.i18n.json

```diff
@@ -2011,6 +2011,7 @@
   "Unmute_user": "Permitir que o usuário fale",
   "Unnamed": "Sem nome",
   "Unpin_Message": "Desafixar Mensagem ",
+  "Unread": "Mensagens Não Lidas",
```

Comment:
the correct translation is just `Não Lidas` since it is used to show unread rooms, not messages.

Reply:
Fixed it.